src/test/java/Elements/RoomDisplayed_Elements.java

```diff
@@ -0,0 +1,21 @@
+package Elements;
+
+import org.openqa.selenium.By;
+
+public class RoomDisplayed_Elements {
```

Comment:
elelements are created per page not per test scenario. call this Room Elements

Reply:
Done

Reply:
rename